regression/contracts-dfcc/assigns_replace_havoc_dependent_targets_pass/vect.h

```diff
@@ -11,8 +11,8 @@ void resize_vec(vect *v, size_t incr)
   // clang-format off
 __CPROVER_requires(
   __CPROVER_is_fresh(v, sizeof(vect)) &&
+  0 < v->size && v->size < __CPROVER_max_malloc_size &&
```

Comment:
IMHO, It seems counter-productive to ask users to add an extra assumption in their contracts to make sure the size will be smaller than the maximum malloc size for CBMC. This is outside of the scope of this PR, but we should have a separate discussion why this is necessary in general.

Reply:
I'm with Felipe on this one. Within CBMC if it is fresh it must also be within that size because malloc can only produce things of that size without failing, so I think world would be a better place if we just moved this into the is_fresh predicate - it can return false if the size is too big.
Also, then when we change the memory model we don't have to change all the assumptions in proofs to get rid of this.